compose/swarm.md

```diff
@@ -6,6 +6,13 @@ keywords: documentation, docs,  docker, compose, orchestration, containers, swar
 title: Use Compose with Swarm
 ---
 
+> **Note:** “Swarm” here refers to [Docker Swarm](/swarm/overview.md), a product
+> separate from Docker Engine. It does _not_ refer to [swarm mode](/engine/swarm),
+> which is a built-in feature of Docker Engine introduced in version 1.12.
+>
+> Integration between Compose and swarm mode is at the experimental stage. See
+> [Docker Stacks and Bundles](bundles.md) for details.
```

Comment:
We may want to refer to the `docker stack deploy --compose-file` now, but I kept the original text here